src/main/java/core/basesyntax/reportservices/ReportToFileImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.reportservices;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+
+public class ReportToFileImpl implements ReportToFile {
+
+    @Override
+    public void writeReportToFile(List<String> report, String fileName) {
+        try (PrintWriter writer = new PrintWriter(new FileWriter(fileName))) {
```

Comment:
Use try-with-resources to ensure that the FileWriter is properly closed in case of an exception.